docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-automatic.md

```diff
@@ -59,10 +70,56 @@ For information on monitoring xCluster replication, refer to [Monitor xCluster](
 
 {{% readfile "includes/transactional-remove-db.md" %}}
 
+{{< warning title="Warning" >}}
+
+If you want the databases being removed from replication on the target
+to be fully usable after dropping replication, you need to stop your
+workload (including performing DDLs) to them and wait for the
+replication lag to reach zero before dropping the replication group.
+
+If you stop only running DDLs but not the rest of your workload and wait
+for the DDLs to drain then the target databases will be usable but each
+tablet may be stale by a different amount.  This contrasts with the
+stopping and waiting fully case where all the tables will be up-to-date.
+
+Finally, if you take no precautions then the target databases may
+contain unreadable rows; we recommend dropping such databases rather
+than attempting to use them.
```

Comment:
Lets instead ask them to use the Failover procedure, which will put it in a usable state.

Reply:
If you want the databases being removed from replication on the target
to be fully usable after dropping replication, you need to stop your
workload (including performing DDLs) to them and wait for the
replication lag to reach zero before dropping the replication group. Or you can follow the Failover workflow to ensure the target cuts over to a consistent time.

If you take no precautions then the target databases may
contain unreadable rows; we recommend dropping such databases rather
than attempting to use them.